library/std/build.rs

```diff
@@ -94,7 +94,7 @@ fn main() {
         // Unsupported <https://github.com/llvm/llvm-project/issues/94434>
         ("arm64ec", _) => false,
         // MinGW ABI bugs <https://gcc.gnu.org/bugzilla/show_bug.cgi?id=115054>
-        ("x86", "windows") => false,
+        ("x86_64", "windows") => false,
```

Comment:
Is the ABI bug not present on 32-bit x86, i.e. `x86` tests on `windows` would pass if symbols were available?

Reply:
Great, thanks for confirming. That should be testable once builtins is working then.